package.json

```diff
@@ -1,12 +1,12 @@
 {
   "type": "module",
   "devDependencies": {
-    "@eslint/js": "^9.9.1",
-    "@stylistic/eslint-plugin-js": "^2.6.4",
-    "@types/node": "^22.5.5",
-    "@typescript-eslint/parser": "^8.3.0",
-    "eslint-plugin-jsdoc": "^50.2.2",
-    "prettier": "3.2.5",
-    "typescript-eslint": "^8.3.0"
+    "@eslint/js": "",
```

Comment:
About this commit: yes we haven't carefully analyzed compatibility constraints but we know things work with the given versions.
If you do not specify versions, there is a risk that an update of a random package breaks the CI unrelatedly to the PR

Reply:
Ok, I'll leave the versions in. I am bothered by how we have a bunch of package.json files with differing versions for the same dependencies strewn aroung the repo.